### PR TITLE
Ot90 661

### PR DESCRIPTION
**Pull request by verteramofederico**
endpoints created.

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
-/* GET home page. */
+
+const news = require('../components/news/network')
+/* const newsAuthMiddleware = require('../middlewares/newsAuthMiddleware') */
+
 router.get('/', (req, res) => {
   res.render('index', { title: 'Express' })
 })
 
+router.use('/news', /* newsAuthMiddleware, */ news)
```
Comment (andresmonsa) [inline routes/index.js]: I suggest not including comments between the code

File: routes/index.js [hunk 2]
```diff
@@ -1,9 +1,14 @@
 const express = require('express')
 
 const router = express.Router()
-/* GET home page. */
+
+const news = require('../components/news/network')
+/* const newsAuthMiddleware = require('../middlewares/newsAuthMiddleware') */
```
Comment (andresmonsa) [inline routes/index.js]: I suggest not including comments between the code

**Review by andresmonsa — COMMENTED** [review]
Good job!

File: routes/index.js (hunk 1 above)
Comment (dvelezroman) [reply routes/index.js]: Agree with @andresmonsa  .. no comments within code

**Review by dvelezroman — CHANGES_REQUESTED** [review]
No comments in code... if some code will be needed after... please add a TODO tag above that commented code.

**Comment by verteramofederico** [comment]
comments deleted

**Review by dvelezroman — APPROVED** [review]

**Review by maxipaci — APPROVED** [review]
Nice work

**Review by benjaminPla — APPROVED** [review]
seens nice!

**Pull request merged**